### DISCUSSION TITLE
Use `var(--font-monospace)` for Notification balloon

### DISCUSSION THREAD

**Discussion by ras0q** [Taitava/obsidian-shellcommands#429] [Ideas - done & released]
Currently notice text enclosed in `<code>` uses monospace font
It would be very nice to be able to use the font set by Obsidian!

![image](https://github.com/user-attachments/assets/ced4bc30-5a94-4289-a9c7-8e763d0feed3)

**Comment by Taitava** [comment]
Hi! Sorry for the delay.

There's actually a setting for this, (but I haven't written documentation for it, issue #329). 

- From Obsidian's settings panel, please go to: _Community plugins_ | _Shell commands_ | _Output_ tab.
- There you can disable the setting _Outputting to Notification/error balloon uses monospace formatting_.

Please let me know if this helps? 🙂 

![kuva](https://github.com/user-attachments/assets/8491252e-121c-4254-adc9-b6b0eed82462)

**Comment by Taitava** [comment]
Or did I understand incorrectly? Are you willing to keep the monospace, but just use a differerent font?

**Reply by ras0q** [reply]
I have set up a monospace font and would like this font to apply this font when formatting!
![image](https://github.com/user-attachments/assets/8864f77a-43ae-4729-8c21-2d76d92b29ee)

**Reply by Taitava** [reply]
Ok, I can do it. I created an issue for it: #431 

Thank you for your suggestion! 🙂

**Reply by Taitava** [reply]
Hi @ras0q ! This is now released in SC `0.23.0`.

Please let me know if you have any feedback. 🙂

**Reply by ras0q** [reply]
Perfect! Thank you!
